package.json

```diff
@@ -15,6 +15,7 @@
 		"eslint": "^6.8.0",
 		"find-java-home": "^1.1.0",
 		"globby": "^11.0.0",
+		"html-escaper": "^3.0.0",
```

Comment:
This was previously a transitive dependency. If we add it as a direct dependency, we will need to open a 3pp.